site/content/docs/4.3/components/card.md

```diff
@@ -4,6 +4,9 @@ title: Cards
 description: Bootstrap's cards provide a flexible and extensible content container with multiple variants and options.
 group: components
 toc: true
+external_js:
+  - src: "https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js"
```

Comment:
perhaps allow passing other attributes like `async` might be good, like in this case.

Reply:
I think we still have to add `async: true` to card.md for full effect in example layout